code/modules/power/apc.dm

```diff
@@ -809,14 +807,15 @@
 				else
 					to_chat(H, "<span class='warning'>You can't receive charge from the APC!</span>")
 			return
-		if(H.a_intent == INTENT_GRAB)
+		if((H.a_intent == INTENT_GRAB) && (drain_stop < world.time -75))
 			if(cell.charge == cell.maxcharge)
 				to_chat(H, "<span class='warning'>The APC is full!</span>")
 				return
 			var/obj/item/organ/stomach/ethereal/stomach = H.getorganslot(ORGAN_SLOT_STOMACH)
 			if(stomach.crystal_charge < 10)
 				to_chat(H, "<span class='warning'>Your charge is too low!</span>")
 				return
+			drain_stop = world.time
```

Comment:
why not just add 75 here, and then check it's drain_stop < world time in the if checks?

Reply:
I mean I suppose I could if you want me to, it'd be super minimally easier but like sort of seems pointless now since they do the same thing. ill change it tho if u want

Reply:
I want you to do it